test-project/Assets/EditmodeTests/Subscriptions/RequireablesDisableTests.cs

```diff
@@ -179,6 +241,12 @@ private class MultipleReaderBehaviour : MonoBehaviour
             [Require] public PositionReader PositionReader;
             [Require] public TestCommandsReader TestCommandsReader;
         }
+        
+        private class MultipleComponentBehaviour : MonoBehaviour
```

Comment:
how is this behaviour different to the one above? both have component readers?

Reply:
The above one has commands, I wanted to ensure its just components. with nothing on them